lib/internal/Magento/Framework/App/Test/Unit/FrontControllerTest.php

```diff
@@ -55,7 +87,7 @@ public function testDispatchThrowException()
     {
         $validCounter = 0;
         $callbackValid = function () use (&$validCounter) {
-            return $validCounter++%10 ? false : true;
+            return ($validCounter + 1) % 10 ? false : true;
```

Comment:
Looks like it will be always false, please double check it

Reply:
@ihor-sviziev thank you. Updated.